db/models/users.js

```diff
@@ -5,7 +5,10 @@ module.exports = (sequelize, DataTypes) => (
     last_attempt_verify_email: DataTypes.DATE,
     phone_number: DataTypes.STRING,
     phone_number_is_verified: DataTypes.BOOLEAN,
-    phone_code_attempts: DataTypes.INTEGER,
+    phone_code_attempts: {
```

Comment:
Because the DB is already in existence, changing the model here might cause trouble.

Reply:
NVM, this is correct.